### DISCUSSION TITLE
Configuration for 2 or more exporters

### DISCUSSION THREAD

**Discussion by cristianmc** [akvorado/akvorado#536] [Q&A] [answered]
Hi nice ppl, I need help to configure 2 or more exporters with different snmp communities and different interface classifiers, is this possible?

**Comment by vincentbernat** [comment, marked as answer]
The `communities` keyword accept a map from subnets (which can be /32) to community. For the interface classifiers, you have access to Exporter.IP and Exporter.Name to handle this case.

**Reply by cristianmc** [reply]
thanks for your help Vincent, great job by the way with this tool!
snmp communities for each exporter seems to work :
`communities:`
    `131.xxx.xx.254: arlab-noc`
    `131.xxx.xx.201: arlab-snmp`

Could you help me with the exports-classifiers section? is there a way to replace the "Exporter.Name" variable with a real exporter name?
and can you customize the interface-classifiers for each?
If you would be so kind, I would really appreciate it with an example.

**Reply by vincentbernat** [reply]
```yaml
- Exporter.Name == "something" && ClassifyExternal()
- Exporter.Name == "else" && ClassifyInternal()
```

**Reply by cristianmc** [reply]
thank you very much, one more shoot.
Is there a list of other parameters for this section?
```
console:
  homepage-top-widgets: [src-as, src-country, etype]

```

**Comment by vincentbernat** [comment]
On 2023-03-03 17:32, Cristian Sanchez wrote:
> thank you very much, one more shoot.
> Is there a list of other parameters I can use in this section?
> |console: homepage-top-widgets: [src-as, src-country, etype]|

They are listed in the documentation.
